CHANGELOG.md

```diff
@@ -18,6 +18,17 @@
 - Set ip_address to {{auto}} by default, even if sendDefaultPII is disabled ([#1665](https://github.com/getsentry/sentry-dart/pull/1665))
   - Instead use the "Prevent Storing of IP Addresses" option in the "Security & Privacy" project settings on sentry.io
  
```

Comment:
Again, just a generic nit. I think it would make sense to keep `## Unreleased` instead of `## 8.0.0` as likely it will be released as alpha or beta first.

Reply:
the plan is to fork the releaase 8.0.0 again to a 8.0.0 beta branch and release that one if that makes sense

Reply:
but true, if I start a beta release from the v8 branch then it needs to be unreleased